backend/molgenis-emx2-sql/src/main/java/org/molgenis/emx2/sql/SqlDatabase.java

```diff
@@ -395,15 +402,20 @@ public void grantCreateSchema(String user) {
 
   @Override
   public void setActiveUser(String username) {
+    if (username == null) {
```

Comment:
is empty String a valid userName ? , suggest to add to param validation

Reply:
I agree.